runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -335,6 +335,17 @@ public void onGameTick(final GameTick event)
 			}
 		}
 
+		if (clue instanceof CoordinateClue)
```

Comment:
Add ``|| clue instanceof CoordinateClue`` to the ``if (clue instanceof EmoteClue)`` instead

Reply:
The two inventory checks are different, one is checking the equipped items and the other is the actual inventory. I can do it like this if you really want but I think the way I had it is more readable 
```		
boolean isCoordinateClue = clue instanceof CoordinateClue;
		
if (clue instanceof EmoteClue || isCoordinateClue)
{
	ItemContainer container = client.getItemContainer(isCoordinateClue? InventoryID.INVENTORY : InventoryID.EQUIPMENT);
			
	if (container != null)
	{
		equippedItems = container.getItems();
	}
}```